manifests/repo.pp

```diff
@@ -3,6 +3,7 @@
 class jenkins::repo (
   Stdlib::Httpurl $base_url = 'https://pkg.jenkins.io',
   String $gpg_key_filename = 'jenkins.io.key',
+  Integer $enabled = 1,
```

Comment:
Pretty sure `yumrepo` also accepts a `Boolean`, which I think is cleaner.

Reply:
Of course, just a brainfog moment.

Reply:
I'm not that familiar with suse/debian, but I checked the .pp and currently no enable variable was available, but if it is possible to add the toggle to zypprepo or apt::source I won't be against it.